dartagnan/src/main/java/com/dat3m/dartagnan/GlobalSettings.java

```diff
@@ -53,4 +57,30 @@ public static void LogGlobalSettings() {
         logger.info("REFINEMENT_GENERATE_GRAPHVIZ_DEBUG_FILES: " + REFINEMENT_GENERATE_GRAPHVIZ_DEBUG_FILES);
         logger.info("REFINEMENT_SYMMETRY_LEARNING: " + REFINEMENT_SYMMETRY_LEARNING.name());
     }
+
+    public static String getOrCreateOutputDirectory() throws IOException {
+        String path = getOutputDirectory();
+        Files.createDirectories(Paths.get(path));
+        return path;
+    }
+
+    public static String getOutputDirectory() {
+        if (isJUnitTest()) {
+            return "target/output";
+        }
+        String env = System.getenv("DAT3M_OUTPUT");
```

Comment:
In my opinion, it would be cleaner to use a config file instead of an environment variable to read output directory. But this would require changes in many places (e.g., vsyncer), so I will skip it for now.